e2e/e2etests/test_whitelist_asset.go

```diff
@@ -18,8 +18,8 @@ import (
 	crosschaintypes "github.com/zeta-chain/node/x/crosschain/types"
 )
 
-// TestWhitelistERC20 tests the whitelist ERC20 functionality
-func TestWhitelistERC20(r *runner.E2ERunner, _ []string) {
+// TestWhitelistAsset tests the whitelist asset functionality
```

Comment:
i think keeping whitelist erc20 test name is more consistent since erc20 is being whitelisted using whitelist asset, same as solana test is whitelist spl